client/src/components/profileInfoEdit/ProfileClearModal.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import toast from 'react-hot-toast';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { clearTeam, deleteGHPages } from '../../actions/team';
+
+const ProfileClearModal = ({ clearAlert, setclearAlert }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const HandleClear = () => {
+    const access_token = localStorage.getItem('GH_access_token');
+    if (access_token === null) {
+      toast.error('Log in with github account');
+    } else {
+      try {
+        dispatch(deleteGHPages(teamId, access_token));
+      } catch (error) {
+        toast.error('GitHub Pages doesnt exist');
+      }
+      dispatch(clearTeam(teamId));
+      toast.success('profile data has been successfully cleared');
```

Comment:
I think it's better to show a success toast within the dispatch action instead of using a generic toast here